modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -0,0 +1,389 @@
+|`otlp`, `jaeger`, `zipkin`
+|None
+
+|processors:
+|Processors run through the data between it is received and exported. By default, no processors are enabled.
+|
+|None
+
+|exporters:
+|An exporter sends data to one or more back ends or destinations. By default, no exporters are configured. There must be at least one enabled exporter for a configuration to be considered valid. Exporters are enabled by being added to a pipeline. Exporters might be used with their default settings, but many require configuration to specify at least the destination and security settings.
+|`otlp`, `otlphttp`, `jaeger`, `logging`, `prometheus`
+|None
+
+|service:
+  pipelines:
+|Components are enabled by adding them to a pipeline under `services.pipeline`.
+|
+|
+
+|service:
+  pipelines:
+    traces:
+      receivers:
+|You enable receivers for tracing by adding them under `service.pipelines.traces`.
+|
+|None
+
+|service:
+  pipelines:
+    traces:
+      processors:
+|You enable processors for tracing by adding them under `service.pipelines.traces`.
+|
+|None
+
+|service:
+  pipelines:
+    traces:
+      exporters:
+|You enable exporters for tracing by adding them under `service.pipelines.traces`.
+|
+|None
+
+|service:
+  pipelines:
+    metrics:
+      receivers:
+|You enable receivers for metrics by adding them under `service.pipelines.metrics`.
+|
+|None
+
+|service:
+  pipelines:
+    metrics:
+      processors:
+|You enable processors for metircs by adding them under `service.pipelines.metrics`.
+|
+|None
+
+|service:
+  pipelines:
+    metrics:
+      exporters:
+|You enable exporters for metrics by adding them under `service.pipelines.metrics`.
+|
+|None
+|===
+
+== OpenTelemetry Collector components
+
+=== Receivers
+
+==== OTLP Receiver
+
+The OTLP receiver ingests data using the OpenTelemetry protocol (OTLP).
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces, metrics
+
+.OpenTelemetry Collector custom resource with an enabled OTLP receiver
+[source,yaml]
+----
+  config: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:4317 <1>
+          http:
+            endpoint: 0.0.0.0:4318 <2>
+          tls: <3>
+            ca_file: ca.pem
+            cert_file: cert.pem
+            key_file: key.pem
+
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+        metrics:
+          receivers: [otlp]
+----
+<1> The OTLP gRPC endpoint. If omitted, the default `+0.0.0.0:4317+` is used.
+<2> The OTLP HTTP endpoint. If omitted, the default `+0.0.0.0:4318+` is used.
+<3> The TLS server side configuration. Defines paths to TLS certificates. If omitted TLS is disabled.
+
+==== Jaeger Receiver
```

Comment:
[Anchoring in module files](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files)
Missing anchor IDs

Reply:
Thank you for the link. All anchor IDs are now added in this file. I will check the other files too.